controller/plugin_controller.go

```diff
@@ -0,0 +1,233 @@
+package controller
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/wantedly/kong-oauth-token-generator/kong"
+	"github.com/wantedly/kong-oauth-token-generator/model/api"
+	"github.com/wantedly/kong-oauth-token-generator/model/oauth2"
+	"github.com/wantedly/kong-oauth-token-generator/model/plugin"
+)
+
+type PluginController struct {
+	Client *kong.Client
+}
+
+type newPluginFormBody struct {
+	Name string `form:"name" json:"name" binding:"required"`
+}
+
+func parsePluginSchema(c *gin.Context, schema *kong.PluginSchema, prefix string, form map[string]interface{}) {
+	for key, field := range schema.Fields {
+		var (
```

Comment:
To avoid type assertion errors.
```
$ make build
# github.com/wantedly/kong-oauth-token-generator/controller
controller/plugin_controller.go:44: cannot use field.Default (type interface {}) as type string in assignment: need type assertion
controller/plugin_controller.go:50: cannot use field.Default (type interface {}) as type string in assignment: need type assertion
make: *** [build] Error 2
```